src/ghvmctl

```diff
@@ -10,6 +10,13 @@ VM_CPUS="${VM_CPUS:=1}"
 VM_MEM_GIB="${VM_MEM_GIB:=6}"
 VM_DISK_GIB="${VM_DISK_GIB:=12}"
 
+REAL_HOME=${SNAP_REAL_HOME:-$PWD}
```

Comment:
This isn't a good idea, `$PWD` could easily be outside of `$HOME` and thus mean the snap can't write to it.

Reply:
Well, `SNAP_REAL_HOME` is always configured when running in snap, as the comment mentions this is meant to be used only when running as a script outside the snap for development purposes.

However, if you prefer this can be moved inside the `SNAP` check below so we're even more sure that it won't affect the `SNAP` case (unless one plays dirty with `snap run --shell`).

Reply:
Ah yeh, this was a mistake on my part. This can probably stay as it is - sorry!